README.cs.md

```diff
@@ -70,20 +72,23 @@ podporovány sway (je známo, že gdm funguje docela dobře).
 [ja]: README.ja.md
 [ko]: README.ko.md
 [nl]: README.nl.md
+[no]: README.no.md
 [pl]: README.pl.md
 [pt]: README.pt.md
 [ro]: README.ro.md
 [ru]: README.ru.md
+[sv]: README.sv.md
 [tr]: README.tr.md
 [uk]: README.uk.md
 [zh-CN]: README.zh-CN.md
 [zh-TW]: README.zh-TW.md
 [i3]: https://i3wm.org/
 [Wayland]: http://wayland.freedesktop.org/
 [FAQ]: https://github.com/swaywm/sway/wiki
-[IRC channel]: https://web.libera.chat/gamja/?channels=#sway
+[IRC kanál]: https://web.libera.chat/gamja/?channels=#sway
 [E88F5E48]: https://keys.openpgp.org/search?q=34FF9526CFEF0E97A340E2E40FDE7BE0E88F5E48
-[GitHub releases]: https://github.com/swaywm/sway/releases
-[Development setup]: https://github.com/swaywm/sway/wiki/Development-Setup
+[Vydání na GitHubu]: https://github.com/swaywm/sway/releases
+[Připravení vývojového prostředí]: https://github.com/swaywm/sway/wiki/Development-Setup
 [wlroots]: https://gitlab.freedesktop.org/wlroots/wlroots
+[swaybg]: https://github.com/swaywm/swaybg/
 [scdoc]: https://git.sr.ht/~sircmpwn/scdoc
```

Comment:
Translating these will break links I'm afraid.

Reply:
my bad, fixed it